Map/Source/Map.cpp

```diff
@@ -0,0 +1,6 @@
+#include "Map.hpp"

```

Comment:
ten plik jest zbedny, nie tworzymy pliku cpp dla interfejsu.
To Co zrobiles powinno byc w klasie SquareMap